### ISSUE TITLE
SSSP Starves the Mesos cluster's resources

### ISSUE THREAD

**Issue by stelcheck** [mesosphere-backup/sssp#8]
SSSP does not respect the configuration and starts using all resources on the cluster for some reason.

**Comment by solidsnack** [comment]
Do you mean, it just keeps spawning and spawning executors?

**Comment by stelcheck** [comment]
No, I mean the framework and its executors just takes all available resources instead of what is configured.

**Comment by py-man** [comment]
I am seeing the same issue - 6 slave nodes, 10CPU 16GB memory available, All consumed as Offers - SSSP works corrects , however Marathon can no longer get any offers - all offers held by the slave nodes and SSSp framework.

**Comment by devinsba** [comment]
The offers aren't being declined in the resourceOffers callback in the scheduler. You have to explicitly decline offers that aren't going to be used. I would submit a pull request but I'm not a scala dev so it would be best if someone who actually writes it fixed it

**Comment by ror6ax** [comment]
Hi, I know it's sort of necroposting, but is there any action happening here?
